webapp-client/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "parser": "babel-eslint",
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true
+  },
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/destructuring-assignment": "warn",
+    "spaced-comment": ["error", "always"]
```

Comment:
Nice! I think there are a few more fix ones that might be useful but hopefully they are included in prettier so that there's no need to add here. For example:
no-extra-parens
no-else-return
array-bracket-spacing
eol-last (helps fix the new line at end of file!)
no-trailing-spaces
quotes
space-in-parens
no-var
prefer-const